metrics/ci_integrations/test/source/github_actions/client_factory/github_actions_source_client_factory_test.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:ci_integration/client/github_actions/github_actions_client.dart';
+import 'package:ci_integration/source/github_actions/client_factory/github_actions_source_client_factory.dart';
+import 'package:ci_integration/source/github_actions/config/model/github_actions_source_config.dart';
+import 'package:ci_integration/util/authorization/authorization.dart';
+import 'package:test/test.dart';
+
+import '../test_utils/test_data/github_actions_config_test_data.dart';
+
+// https://github.com/platform-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors, avoid_redundant_argument_values
+
+void main() {
+  final githubActionsSourceConfig = GithubActionsConfigTestData.sourceConfig;
+  final githubActionsSourceClientFactory = GithubActionsSourceClientFactory();
+
+  GithubActionsClient createClient(GithubActionsSourceConfig config) {
```

Comment:
Should we inline this function to make test cases more clear? It seems like we can move this code to the `setUp` callback.